owasp-top10-2017-apps/a10/games-irados/docs/ATTACK.md

```diff
@@ -0,0 +1,61 @@
+# Attack Narrative - GamesIRADOS
+The main goal of this document is to describe how important it is to properly log all requests made to the application and how easily malicious requests could go unnoticed.
+
+If you don't know [secDevLabs] or this [intended vulnerable web application][2] yet, you should check them out before reading this narrative.
+
+---
+## 👀
+
+With the goal of verifying how an application handles events that are considered malicious, two attacks will be done to test it:
+* Brute forcing the login screen
+* Brute forcing the coupon validation screen
+
+Initially, we begin by performing a wrong login attempt to see what the application does in that case, as shown by the image below:
+
+<p align="center">
+    <img src="attack1.png"/>
+</p>
+
+## 🔥
+
+Now, using [Burp Suite] as a proxy, we can use it to fire as many requests as we want, only changing the content of the `password` field at every request, as depicted by the images below:
+
+<p align="center">
+    <img src="attack2.png"/>
+</p>
+
+<p align="center">
+    <img src="attack3.png"/>
+</p>
+
+By having a look at the application on the server side, it's possible to see how little it logs our attack attempt, as shown below:
```

Comment:
What do you think of changing `it's possible to see how little it logs our attack attempt` into `it is possible to that the logs provide little information regarding the attack`?